flaml/ml.py

```diff
@@ -265,14 +268,13 @@ def sklearn_metric_loss_score(
             used to calculate the metric. E.g., 2d for log_loss and 1d
             for others.
         y_true: A 1d numpy array of the true labels.
-        labels: A 1d numpy array of the unique labels.
+        label_list: A list or an array of the unique labels.

```

Comment:
don't use "_list" because it may not be a list.
"labels" is OK because it's the conventional variable name.